ui/desktop/app/controllers/scopes/scope/projects/targets/index.js

```diff
@@ -100,4 +108,37 @@ export default class ScopesScopeProjectsTargetsIndexController extends Controlle
       session_id
     );
   }
+
+  /**
```

Comment:
None blocking and not mandatory: Would be nice to have a comment like in the other methods, perhaps something like sets freeSearch value and triggers the search? you know better the context. Thoughts?

Reply:
Updated. A lot is going on so comments should be added to each action so we are fully aware of what is going on.